src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -394,7 +409,8 @@ public void deleteCommentsByStudentEmail(String courseId, String email) {
         
         // student right now cannot be giver, so no need to&should not check for giver
         List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId, 
-                CommentRecipientType.PERSON, email);
+                                                                               CommentParticipantType.PERSON,
+                                                                               email);
```

Comment:
Alternatively, something like this is still ok (it saves one line without reducing readability).
```
List<Comment> recipientComments = this.getCommentEntitiesForRecipients(courseId, 
                                                  CommentRecipientType.PERSON, email);
```
The second line starts on the right side of the `.` in the line above, so the hieararchy is still maintained.